model/course.go

```diff
@@ -69,6 +69,16 @@ func (c Course) IsLive() bool {
 	return false
 }
 
+// IsNextLectureStartingSoon checks whether the course has a lecture that starts soon
+func (c Course) IsNextLectureStartingSoon() bool {
+	for _, s := range c.Streams {
```

Comment:
You can use the courses `getNextLecture` method I think

Reply:
Ah, that's way cleaner!

Reply:
`getNextLecture` was broken, but it know should be fixed